### ISSUE TITLE
Missing TS declarations '@bugsnag/react-native' ^7.3.1

### ISSUE THREAD

**Issue by teo029** [bugsnag/bugsnag-react-native#477]
TS error: 

```Could not find a declaration file for module '@bugsnag/react-native'. '.../node_modules/@bugsnag/react-native/src/notifier.js' implicitly has an 'any' type.```

There seem to be missing @bugsnag/react-native declarations.

Version: `"@bugsnag/react-native": "^7.3.1"`,

**Comment by steve-nester-uk** [comment]
@teo029 can you describe the steps to replicate please, including the steps you took for installation, e.g. 

```
npm install --save @bugsnag/react-native
Added import Bugsnag from '@bugsnag/react-native' to App.js
etc
```

**Comment by teo029** [comment]
The steps I took:

1. `npm install --save @bugsnag/react-native`
2. Added `import Bugsnag from '@bugsnag/react-native'` in App.js

It triggered the following TS error:

`Could not find a declaration file for module '@bugsnag/react-native'. '<MY_PROJECT_PATH>/node_modules/@bugsnag/react-native/src/notifier.js' implicitly has an 'any' type.`

**Comment by johnkiely1** [comment]
Hi @teo029.
Thanks for the extra info and that for reporting this. We are looking into creating a fix for this asap. We will update you as soon as it is ready.

**Comment by steve-nester-uk** [comment]
Fixed in [v7.3.2](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.3.2)
